packages/ra-ui-materialui/src/field/sanitizeFieldRestProps.ts

```diff
@@ -1,4 +1,5 @@
 export const sanitizeFieldRestProps: (props: any) => any = ({
+    addLabel,
```

Comment:
This one shouldn't be there - `addLabel` was removed in 4.0. If we still set it, that's a mistake.